docs/markdown/Writing Plugins/common-plugin-tasks/plugin-helpers.md

```diff
@@ -1,28 +0,0 @@
----
```

Comment:
I think you need to delete this in Readme directly, if you haven't yet.

Reply:
Only in 2.18, which we haven't cut yet. But good reminder!

Reply:
Ugh we really need to move off readme.com